.github/workflows/execute_tutorial_notebooks_in_external_repo.yml

```diff
@@ -0,0 +1,88 @@
+########################################################################################
+# Trigger the notebooks' execution in the external repository.
+#
+# If the "main" branch triggered this event, then the nb execution is triggered in the
+# "main" branch, otherwise in the dev branch.
```

Comment:
I really like this idea! So, did I understand correctly that all branches but master will automatically render into "dev"?

Reply:
Yes. However, right now only the main branch is pull in the examples repo. I will add something to pull the same branch that sent the triggering commit to the external repo. In that way, we can test if the examples execute correctly with development branches.